docs/Concepts/Privacy/Private-Transactions.md

```diff
@@ -61,6 +61,24 @@ different to the nonce for account A for privacy group AB.
 
 The nonce for the [privacy marker transaction] is the public nonce for the account.
 
+### Nonce validation 
+
+Unlike public transactions, private transactions are not submitted to the transaction pool. The private
+transaction is distributed directly to the participants in the transaction and the privacy marker
+transaction is submitted to the [transaction pool](../Transactions/Transaction-Pool.md). 
+
+Unlike [public transaction nonces](../Transactions/Transaction-Validation.md), private transaction
+nonces are not validated when the private transaction is submitted. Private transactions are directly
```

Comment:
I'd probably remove the `Private transactions are directly distributed` because we already said that in the previous paragraph.

Reply:
Done